daemon/src/lib.rs

```diff
@@ -85,7 +86,7 @@ where
             Box<dyn MessageChannel<NewTakerOnline>>,
             Box<dyn MessageChannel<FromTaker>>,
         ) -> T,
-        settlement_time_interval_hours: time::Duration,
+        settlement_interval: time::Duration,
```

Comment:
I know the name is ugly but it would be great to minimize the diff of this PR to just the auto-roll over feature. Can you please undo this rename and squash all commits afterwards?

Feel free to send a follow-up PR that cleans up these names.

Reply:
For future PRs, can you please quickly scan the entire diff of the PR yourself just before you request a review? :)
These things are easy to spot!

Reply:
sorry thats my bad